kit/Kit.hpp

```diff
@@ -186,6 +186,43 @@ class KitSocketPoll final : public SocketPoll
 #endif
 };
 
+class LogUiCmd
```

Comment:
Can we put this UI command logging stuff in it's own header somewhere ?

Reply:
moved LogUiCmd from Kit.hpp/.cpp to logUI.hpp/.cpp